db/migrate/20231108210613_change_associations_duration_rates.rb

```diff
@@ -0,0 +1,19 @@
+class ChangeAssociationsDurationRates < ActiveRecord::Migration[7.0]
```

Comment:
Please add frozen string literal magic comment

Reply:
I'll have to add a sticky note somewhere so I don't keep forgetting to do it 🤦‍♀️

Reply:
we should set up rubocop to help us remember things like this